scala/loadadw/src/main/scala/example/checkRemoteBucket.scala

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Do we need separate PR for this?

Reply:
It was part of the prior PR, related to RP